.github/workflows/ci.yml

```diff
@@ -39,7 +39,15 @@ jobs:
             **/setup.cfg
       - run: pip install -U pip wheel setuptools
       - run: pip install -U tox tox-gh-actions
-      - run: tox
+      - name: "Run tox targets for ${{ matrix.python-version }}"
+        env:
+          TOX_SKIP_ENV: ".*djangomain.*"
+        run: "python -m tox"
+      - name: "Run tox targets for ${{ matrix.python-version }} for django main"
+        env:
+          TOX_SKIP_ENV: ".*django[^m].*"
+        run: "python -m tox"
+        continue-on-error: true
```

Comment:
We could want to do this using the GHA matrix instead so there's an additional factor `djangomain` that's only run with Python 3.11?

Reply:
Yes, I'm looking at that :) I you have a quick way to do that let me know :)

Reply:
I guess 
```yaml
        include:
          - python-version: "3.8"
          - python-version: "3.9"
          - python-version: "3.10"
          - python-version: "3.11"
          - python-version: "3.11"
            djangomain: true
```
would do the trick, and then `if: ${{ matrix.djangomain }}`?